Src/core/controller/PostController.php

```diff
@@ -1,8 +1,8 @@
 <?php
-namespace core\controller;
+namespace Aot\core\controller;
```

Comment:
`core` keyword is a folder where components are generic. It means they can be reuse for another project (App, Router...). Can u remove this folder from path please?

Reply:
done